modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/index.tsx

```diff
@@ -43,7 +43,11 @@ type OutletContext = {
 		mergedSubtaskNames: string;
 		splitSubtaskNames: string;
 		subtaskIssues: TestraySubTaskIssue[];
-		testraySubtask: TestraySubTask;
+		testraySubtask: TestraySubTask & {
+			actions: {
+				[key: string]: string;
+			};
+		};
```

Comment:
@rafaelfs98 add in the type `TestraySubtask` this type: `actions: ObjectActionsItems`

See this example https://github.com/pt-liferay-solutions/liferay-portal/blob/3e876b037049874d545a8184acfb3b3461d6c59c/modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/types.ts#L161